src/nbody_simulation_result.jl

```diff
@@ -211,9 +211,9 @@ function electrostatic_potential(p::ElectrostaticParameters, indxs::Vector{<:Int
             if !in(j, exclude[i])
                 rj = @SVector [rs[1, j], rs[2, j], rs[3, j]]
 
-                (rij, rij_2, success) = apply_boundary_conditions!(ri, rj, pbc, p.R2)
-                if !success
-                    rij = p.R
+                (rij, r, r2) = get_interparticle_distance(ri, rj, pbc)
+                if !(r2 < p.R2)
+                    rij = p.R # NOTE: hmm?
```

Comment:
?

Reply:
Sorry, forgot to delete a temporary note for debugging. Disregard, it seems to work fine.